src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageSolutionDetailControlModel.cs

```diff
@@ -120,6 +121,7 @@ public string InstalledVersions
         private async Task UpdateInstalledVersionsAsync(CancellationToken cancellationToken)
         {
             var hash = new HashSet<NuGetVersion>();
```

Comment:
It's worth updating this variable to distinguish from the new set we created for vulnerabilities and make the name reflect its role.